core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/dot-experiments-configuration.component.ts

```diff
@@ -100,8 +102,11 @@ export class DotExperimentsConfigurationComponent implements OnInit {
      * @returns void
```

Comment:
fix doc

Reply:
Its already fixed

Reply:
no, you're missing `experimentId: string` in `deleteVariant` method